x-pack/plugins/timelines/public/components/t_grid/body/index.tsx

```diff
@@ -584,7 +584,7 @@ export const BodyComponent = React.memo<StatefulBodyProps>(
       direction: 'asc' | 'desc';
     }> = useMemo(
       () =>
-        sort.map((x) => ({
+        sort?.map((x) => ({
```

Comment:
Hey `sort` has type `SortColumnTimeline[]`. Looks like it always array, maybe we shoulde change a type or remove optional chaining?

Reply:
ah yeah I was debugging something where the sort could be undefined for some reason. I think it had to do with debugging issues in the preview back when I was trying to get that sorted out. I'll revert the change. Thanks!